src/components/v5/common/Modals/ManageMemberModal/ManageMemberModal.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, useState } from 'react';
+import { useIntl } from 'react-intl';
+
+import Modal from '~v5/shared/Modal';
+import MembersSelect from '~v5/shared/MembersSelect';
+import Select from '~v5/common/Fields/Select';
+import { manageMemberActions } from './consts';
+import { ManageMemberModalProps } from './types';
+
+const displayName = 'v5.Modals.ManageMemberModal';
```

Comment:
You've missed 'common'
Please, check it in your other components as well

Reply:
Common added